Dev board revisions

mgordon2646 opened the pull request:
Still the same PCB (or is it?)

mgordon2646 committed:
Theseus_Dev_Rev2

It's still the same PCB

mgordon2646 committed:
Minor routing changes

Still the same PCB

Joe-Joe-Joe-Joe commented:
1. Please do not export the gerbers to the top level folder. Specifically, what you should do is install this JLC PCB toolkit plugin. This adds a button to your PCB editor which you just click, and it puts all your gerbers in the specific format JLC wants them, that you can zip and send directly to JLC
<img width="413" height="238" alt="image" src="https://github.com/user-attachments/assets/dacbea2d-1852-45da-a827-945cf8e88f2f" />

3. GND symbols pointing up is a sin
<img width="150" height="139" alt="image" src="https://github.com/user-attachments/assets/2f343cb2-aa64-488e-b288-2eac18c1723d" />

4. I don't know what solenoids you are spec'd for, but your NMOS has a max current of 200mA, and your diodes max out at 500mA. That seems concerningly low.
5. Part number for ferrite bead. Look at processor for what was used there?
6. I'll be very impressed if TC2/TC3 are your own inventions. That being said, seems like TC1 will win in the end. I do not have to pick apart TC2/3 but they seem cool
7. I think Ash is too anal about daisy chaining but this is a good example of where it does actually make a difference between the board working and not working
<img width="220" height="736" alt="image" src="https://github.com/user-attachments/assets/a9b22df8-f2c1-450b-9448-b2a81297bf81" />

8. I am 98% sure the blue diodes we stock are also 1206
<img width="184" height="125" alt="image" src="https://github.com/user-attachments/assets/98be74de-85be-482d-b322-7715eda682e6" />

9. You have infinite layout space please just label every screw terminal. I promise it will save so many headaches down the line
<img width="659" height="453" alt="image" src="https://github.com/user-attachments/assets/f3ce7ab5-616e-472e-86a0-df551a4263aa" />

10. In general it is not good to jump planes, esp ground planes, like this. For reference, a 0.2mm trace is ~62mR/inch, and your PT sense resistor is +/- 150mR. So in this case, probably wont make a difference, but its not nothing. 
<img width="189" height="638" alt="image" src="https://github.com/user-attachments/assets/9a91c8d2-d7e6-46c5-8fd6-2b6ff1fdc9c6" />

11. Please super clearly mark how to orient the programmer on this
<img width="696" height="765" alt="image" src="https://github.com/user-attachments/assets/7f1e47b6-30fb-4dba-be8b-65dd6799240f" />

12. A lot of silkscreen labels are at various rotations. We try to keep all silkscreen labels all pointing the same direction - makes it easier to read

mgordon2646 committed:
Fixes V3

- Copied Ferrite Bead P/N from processor
- Added external oscillator
- Made status LED pins pull-down
- Copied processor programming header

mgordon2646 committed:
NMOS and Diode Update

Updated Diode and NMOS to support 1A max. Fixed valve state LED to draw from BATT instead of MCU.
